sos/report/plugins/ceph_mon.py

```diff
@@ -110,14 +110,12 @@ def setup(self):
             "ceph tell mon.%s mon_status" % mon_id for mon_id in mon_ids
         ], subdir="json_output", tags="insights_ceph_health_detail")
 
-        # these can be cleaned up too but leaving them for safety for now
         self.add_forbidden_path([
             "/etc/ceph/*keyring*",
             "/var/lib/ceph/*keyring*",
             "/var/lib/ceph/*/*keyring*",
             "/var/lib/ceph/*/*/*keyring*",
             "/var/lib/ceph/osd",
-            "/var/lib/ceph/mon",
```

Comment:
Only the `keyring` file(s) contain sensitive info. Since that's explicitly excluded, no need to forbid the whole directory. This would otherwise prevent collecting `/var/lib/ceph/mon/*/kv_backend`.